From: Viresh Kumar <viresh.kumar@linaro.org>
Date: Fri, 22 Apr 2016 16:58:39 +0530
Subject: cpufreq: dt: Mark platdev machines array as __initconst
Git-commit: e92bb16674c5156ea9230ba7a3ab6d490750888f
Patch-mainline: v4.7-rc1
References: fate#322003

The machines array in cpufreq-dt-platdev is used only once at boot time
and so should be marked with __initconst, so that kernel can free up
memory used for it, if required.

Suggested-by: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/cpufreq/cpufreq-dt-platdev.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -11,7 +11,7 @@
 #include <linux/of.h>
 #include <linux/platform_device.h>
 
-static const struct of_device_id machines[] = {
+static const struct of_device_id machines[] __initconst = {
 };
 
 static int __init cpufreq_dt_platdev_init(void)
